### PR TITLE
build: Cleanup Uno.CrossTargetting.props imports

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -41,17 +42,7 @@
...
-		$(UnoTargetFrameworkOverride.ToLowerInvariant().StartsWith('monoandroid'))
-		OR $(UnoTargetFrameworkOverride.ToLowerInvariant().StartsWith('net6.0-android'))
-		OR $(UnoTargetFrameworkOverride.ToLowerInvariant().StartsWith('net7.0-android'))
-		">$(DefineConstants);TARGET_FRAMEWORK_OVERRIDE_ANDROID</DefineConstants>
-
-    <DefineConstants Condition="
-		$(UnoTargetFrameworkOverride.ToLowerInvariant().StartsWith('xamarinios'))
-		OR $(UnoTargetFrameworkOverride.ToLowerInvariant().StartsWith('net6.0-ios'))
-		OR $(UnoTargetFrameworkOverride.ToLowerInvariant().StartsWith('net7.0-ios'))
-		">$(DefineConstants);TARGET_FRAMEWORK_OVERRIDE_IOS</DefineConstants>
```
Comment (Youssef1313) [inline src/Directory.Build.props:54]: `TARGET_FRAMEWORK_OVERRIDE_ANDROID` and `TARGET_FRAMEWORK_OVERRIDE_IOS` are only used in https://github.com/unoplatform/uno/blob/55c0447b1c5367032fa81695f8c2a2684292e6cc/src/SamplesApp/SamplesApp.UITests/Constants.cs#L17-L25

So moved this to the test project, and replaced the two `StartsWith` with a single `EndsWith('-android')`

File: src/Directory.Build.props [hunk 2]
```diff
@@ -3,6 +3,7 @@
   <Import Project="$(SolutionDir)\crosstargeting_override.props" Condition="exists('$(SolutionDir)\crosstargeting_override.props')" />
 
   <Import Project="$(MSBuildSDKExtrasTargets)" Condition="Exists('$(MSBuildSDKExtrasTargets)')" />
+  <Import Project="Uno.CrossTargetting.props" />
```
Comment (Youssef1313) [inline src/Directory.Build.props:6]: Now including this in all projects.

File: src/Uno.CrossTargetting.props [hunk 3]
```diff
@@ -6,9 +6,6 @@
 		<Product>$(AssemblyName) ($(TargetFramework))</Product>
 		<DefaultLanguage>en-US</DefaultLanguage>
 
-		<!-- Version management is now done through NuGet, this avoids issues related version mismatch -->
-		<Version>255.255.255.255</Version>
-
```
Comment (Youssef1313) [inline src/Uno.CrossTargetting.props:11]: Already exists in https://github.com/unoplatform/uno/blob/55c0447b1c5367032fa81695f8c2a2684292e6cc/src/Directory.Build.props#L26-L27

so this seems redundant here.

**Comment by Youssef1313** [comment]
> CSC : error UXAML0001: The type local:TeachingTip could not be found [D:\a\1\s\src\Uno.UI.FluentTheme.v1\Uno.UI.FluentTheme.v1.Skia.csproj::TargetFramework=netstandard2.0]

@jeromelaban Why is this not an error in master?

The type TeachingTip indeed doesn't exist yet.

**Comment by Youssef1313** [comment]
```
/Users/runner/work/1/s/src/SamplesApp/UITests.Shared/Microsoft_UI_Xaml_Controls/RatingControlTests/RatingControlPage.xaml.cs(89,33): error CS0618: 'RatingControl' is obsolete: 'The Windows.UI.Xaml.Controls version of this control is not supported. Please use Microsoft.UI.Xaml.Controls.RatingControl instead.' [/Users/runner/work/1/s/src/SamplesApp/SamplesApp.macOS/SamplesApp.macOS.csproj]
```

Looks like this needs to be fixed. I'll have to verify how the sample behave currently. I expect it to be broken.

**Comment by Youssef1313** [comment]
Hmm... This is actually a weird error. The sample is using `muxcontrols:RatingControl` for all `RatingControl` references, and `muxcontrols` is correctly defined as `xmlns:muxcontrols="using:Microsoft.UI.Xaml.Controls"`

This needs a closer look.

**Comment by jeromelaban** [comment]
This is an issue I've noticed in https://github.com/unoplatform/uno/pull/8074 as well, but I have yet to find the cause.

**Comment by Youssef1313** [comment]
I have to wait for https://github.com/unoplatform/uno/pull/10960 to be merged first. The refactoring of how pages are included will make this PR much easier.

**Comment by Youssef1313** [comment]
Too much conflicts. Re-working this in a new PR.

**Pull request closed without merging**